service.yaml.template

```diff
@@ -17,6 +17,8 @@ environment:
       value: "$$SCONE::password$$"  # get from CAS
     - name: RELEASE
       value: $RELEASE
+    - name: EXCLAIM
```

Comment:
Is this used somewhere? Why add it?

Reply:
Extended the Python program to use this explicitly defined secret